src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,73 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { getUser } from '../../api';
+import { actions } from '../../features/currentTodo';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  selectedTodo: Todo,
+};
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo }) => {
+  const [user, setUser] = useState<User | null>(null);
```

Comment:
The user also should be stored in the global store

Reply:
I thought it shouldn't be stored in the store because we use it only in TodoModal.
But I remake it, so now it's storing in store.